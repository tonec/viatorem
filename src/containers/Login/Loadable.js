import Loadable from 'react-loadable'
import { Loading } from 'components'

let LoadableExample = Loadable({
  loader: () => import('./Login'),
  loading: Loading
})

export default LoadableExample
