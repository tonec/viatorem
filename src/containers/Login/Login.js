import React, { Component } from 'react'
import { OneColumnThin } from 'components'
import LoginFormContainer from './LoginForm/LoginFormContainer'

export class Login extends Component {
  render () {
    return (
      <OneColumnThin title="Login">
        <LoginFormContainer />
      </OneColumnThin>
    )
  }
}

export default Login
