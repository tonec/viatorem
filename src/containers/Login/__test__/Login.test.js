import React from 'react'
import { shallow } from 'enzyme'
import { Login } from '../Login'

describe('Login (snapshot)', () => {
  it('renders according to snapshot when enabled', () => {
    const component = shallow(<Login />)
    expect(component).toMatchSnapshot()
  })
})
