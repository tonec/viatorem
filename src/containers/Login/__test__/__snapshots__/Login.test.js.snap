// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login (snapshot) renders according to snapshot when enabled 1`] = `
<OneColumnThin
  className=""
  offset={9}
  span={6}
  title="Login"
>
  <Connect(ReduxForm) />
</OneColumnThin>
`;
